airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.java

```diff
@@ -79,7 +79,7 @@
 
   private static final long GRACEFUL_SHUTDOWN_MINUTES = 5L;
   private static final int MIN_RECORDS = 500;
-  private static final int BATCH_SIZE = 10000;
+  private static final int BATCH_SIZE = 500_000;
```

Comment:
Since we are using buld upload for postgres and mysql as well, we should increase the batch size. 10K is very small.